classifides/accounts/forms.py

```diff
@@ -0,0 +1,17 @@
+from django import forms
+from django.contrib.auth.models import User
+from django.contrib.auth.forms import UserCreationForm
+
+class CustomUserCreationForm(UserCreationForm):
+    email = forms.EmailField(required=True)
+
+    class Meta:
+        model = User
+        fields = ("username", "email", "password1", "password2")
+
+    def save(self, commit=True):
+        user = super(CustomUserCreationForm, self).save(commit=False)
+        user.email = self.cleaned_data["email"]

```

Comment:
No need to manually set email. Since you have included email in fields, it will automatically be saved in super call.
There is no need to override save method.

Also please add tests for forms and views.

Reply:
ok